feat: clean up all injected class

lekaha opened the pull request:
- Move all injected class (with @Inject annotation) into module where by using @Provider.

lekaha committed:
feat: clean up all injected class

- Move all injected class (with @Inject annotation) into module where by using @Provider.

pokk commented on mobile-ui/src/main/java/com/playone/mobile/ui/injection/module/ActivityBindingModule.kt:
```diff
@@ -1,14 +1,27 @@
 
     @ContributesAndroidInjector(modules =
-        arrayOf(FragmentBindingModule::class))
+        arrayOf(
```
The format was crooked XDD. Please let it fit the coding style. 😀

pokk commented on mobile-ui/src/main/java/com/playone/mobile/ui/injection/module/BrowseActivityModule.kt:
```diff
@@ -15,6 +33,77 @@ import dagger.Provides
+                                                 bufferooRemoteDataStore: BufferooRemoteDataStore)
+            : BufferooDataStoreFactory
+            = BufferooDataStoreFactory(
+            bufferooCache,
```
The format was crooked 😆.

```kotlin
BufferooDataStoreFactory(bufferooCache,
                         bufferooCacheDataStore,
                         bufferooRemoteDataStore)
```

The style will be as like this after we decided.

lekaha replied:
Got trouble on align the parameters.........

pokk commented on mobile-ui/src/main/java/com/playone/mobile/ui/injection/module/BrowseActivityModule.kt:
```diff
@@ -15,6 +33,77 @@ import dagger.Provides
+    @Provides
+    internal fun provideBufferooCacheDataStore(bufferooCache: BufferooCache)
+            : BufferooCacheDataStore
+            = BufferooCacheDataStore(bufferooCache)
```
We might not decide this coding style yet.

```kotlin
// 1
fun foo(xxx: Int)
    : Hoo = Hoo(xxx)

// or

// 2
fun foo(xxx: Int)
    : Hoo =
     Hoo(xxx)
```

Which one is better?

pokk replied:
I prefer the first one. Ofcz, the second line is too long, it will be the second option.

lekaha replied:
I think the same.

lekaha replied:
```
// 0
fun foo(xxx: Int) : Hoo = Hoo(xxx)

If 0 is too long
// 1
fun foo(xxx: Int)
    : Hoo = Hoo(xxx)

If 1 is too long
// 2
fun foo(xxx: Int)
    : Hoo =
     Hoo(xxx)
```

lekaha committed:
Fix https://github.com/lekaha/playone-android/pull/25

- fixed: code style https://github.com/lekaha/playone-android/pull/25#pullrequestreview-98840136

lekaha committed:
fix: code style
